### PR TITLE
#14 / #18 Modify receipt page can be resubmitted.  Bumped up Bb API t…

### PR DESCRIPTION
**Pull request by cbeach47**
…o version 3500.0.0

Ref #14 and #18 

Switched the logic so the parameters of the course and content IDs in `modify_proc.jsp` and sent in the URL, and added some logic to not do a double-save.

v3500.0.0 doesn't look to cause any breaking changes.

### CODE REVIEW

**Comment by cbeach47** [comment]
@abidingotter - a review would be appreciated, but plz don't merge yet.  The Bb sites I use to test are throwing errors when I try to load the WS, so I have a ticket in to fix that, and then I'll run through the flows to ensure v3500.0.0 APIs are playing nice.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ build/
 .project
 .classpath
 .vscode/
+.idea/
```
Comment (ChristianMurphy) [inline .gitignore:14]: :+1:

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,6 @@ build/
...
+**/.editorconfig
```
Comment (ChristianMurphy) [inline .gitignore:15]: :question: is editor config ignore intended?

File: README.md [hunk 3]
```diff
@@ -9,15 +9,13 @@ Version 2.W.V will support Blackboard v3400+
 
 ## Building the primary building block
 ```
```
Comment (ChristianMurphy) [inline README.md]:
```suggestion
```bash
```

File: README.md [hunk 4]
```diff
@@ -9,15 +9,13 @@ Version 2.W.V will support Blackboard v3400+
...
-~$ ./gradlew :oeqPrimaryB2:clean
-~$ ./gradlew :oeqPrimaryB2:buildB2
+~$ ./gradlew :oeqPrimaryB2:clean :oeqPrimaryB2:buildB2
 ```
 The war is placed in (cloned repo)/oeqPrimaryB2/build/libs/
 
 ## Building the primary web service
 ```
```
Comment (ChristianMurphy) [inline README.md]:
```suggestion
```bash
```

File: .gitignore (hunk 2 above)
Comment (cbeach47) [reply .gitignore:15]: For now, yes, it's intended - we should probably match what the primary openEQUELLA repo is doing for styling (google java)

File: gradle.properties [hunk 5]
```diff
@@ -1,4 +1,4 @@
-bbLearnVersion=3400.0.0
+bbLearnVersion=3500.0.0
```
Comment (abidingotter) [inline gradle.properties:1]: So that we can build for any client version of BB, is it possible to alter this via a command line switch?

**Review by abidingotter — APPROVED** [review]
Just one comment from me

File: gradle.properties (hunk 5 above)
Comment (cbeach47) [reply gradle.properties:1]: Yes - via the `-P` argument.  I added some logging and documentation for folks that want to do this.

**Pull request merged**